src/main/java/edu/ie3/datamodel/models/StandardUnits.java

```diff
@@ -89,6 +89,10 @@
   public static final Unit<ThermalConductance> THERMAL_TRANSMISSION = KILOWATT_PER_KELVIN;
   /** Depth of discharge */
   public static final Unit<Dimensionless> DOD = PERCENT;
+  /** Charge/Discharge Rate (constant power) */
+  public static final Unit<Dimensionless> CP_RATE = PU;
```

Comment:
```suggestion
  public static final Unit<DimensionlessRate> CP_RATE = PU_PER_HOUR;
```
Otherwise you wouldn't know, how to handle the value in different time distances.